extras/backup/README.md

```diff
@@ -0,0 +1,130 @@
+# Backup for GKE
+
+This directory contains files essential to demonstrate Backup for GKE service functionality.
+
+For example to migrate from GKE Standart to GKE Autopilot cluster in other region.
+
+## Steps
+
+1. Create GKE Standard as Source cluster
+
+  ```bash
+  gcloud beta container clusters create "source-cluster" \
+    --region="us-central1" \
+    --addons="BackupRestore,HttpLoadBalancing"
+  ```
+
+1. Create GKE Autopilot as Destination cluster
+
+  ```bash
+  gcloud beta container clusters create-auto "destination-cluster" \
+    --region="us-east1"
+  ```
+
+1. Enable Backup for GKE in Destination cluster
+
+  ```bash
+  gcloud container clusters update "${GKE_AUTOPILOT}" \
+    --region="us-east1" \
+    --update-addons="BackupRestore=ENABLED"
+  ```
+
+1. Create External Global IP
+
+  ```bash
+  gcloud compute addresses create "bank-of-anthos" --global
+  ```
+
+1. Deploy the Bank of Anthos to Source cluster
+
+  ```bash
+  kubectl --context="gke_${PROJECT_ID}_us-central1_source-cluster" apply -f extras/jwt
+  kubectl --context="gke_${PROJECT_ID}_us-central1_source-cluster" apply -f kubernetes-manifests/
+  kubectl --context="gke_${PROJECT_ID}_us-central1_source-cluster" apply -f extras/backup/kubernetes-manifests/
```

Comment:
have problems patching the "accounts-db" and "ledger-db" statefulSets because it changes some disallowed spec fields. Delete these two statefulSets before this step:
```shell
kubectl --context="gke_${PROJECT_ID}_us-central1_source-cluster" delete statefulset accounts-db ledger-db
```